crates/iota-types/src/gas_model/gas_v1.rs

```diff
@@ -398,15 +397,17 @@ mod checked {
         /// of the gas meter. We use initial budget, combined with
         /// remaining gas and storage cost to derive computation cost.
         fn summary(&self) -> GasCostSummary {
-            // compute storage rebate, both rebate and non refundable fee
+            // compute computation cost burned and storage rebate, both rebate and non
+            // refundable fee
+            let computation_cost_burned =
```

Comment:
I think here it is needed the flag check, otherwise we loose the backward compatibility with the previously executed transaction effects.